<launch>
  <arg name="input_route" default="input_route"/>
  <arg name="input_scenario" default="input_scenario"/>
  <arg name="input_odometry" default="/localization/kinematic_state"/>
  <arg name="output_trajectory" default="output_trajectory"/>
  <arg name="is_completed" default="is_completed"/>
  <arg name="debug_path_marker" default="debug_path_marker"/>
  <arg name="debug_route_name" default="debug_route_name"/>
  <arg name="diagnostics" default="diagnostics"/>

  <arg name="param_file" default="$(find-pkg-share autoware_static_freespace_planner)/config/static_freespace_planner.param.yaml"/>

  <node pkg="autoware_static_freespace_planner" exec="static_freespace_planner" name="static_freespace_planner" output="screen">
    <remap from="~/input/route" to="$(var input_route)"/>
    <remap from="~/input/scenario" to="$(var input_scenario)"/>
    <remap from="~/input/odometry" to="$(var input_odometry)"/>
    <remap from="~/output/trajectory" to="$(var output_trajectory)"/>
    <remap from="~/debug/path_marker" to="$(var debug_path_marker)"/>
    <remap from="~/debug/route_name" to="$(var debug_route_name)"/>
    <remap from="~/output/is_completed" to="$(var is_completed)"/>
    <remap from="~/output/diagnostics" to="$(var diagnostics)"/>
    <param from="$(var param_file)" allow_substs="true"/>
  </node>
</launch>
